tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +33,13 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert not items
+    assert len(items) == 0
```

Comment:
If the first assert here passes, then we can absolutely assume that the second will as well, so just the first will do!